project/cmd.go

```diff
@@ -19,6 +19,10 @@ func AddCommands(parent *cobra.Command, group *cobra.Group) {
 	project := shared.MakeGroupCommand(group, "project", "Commands for managing projects")
 	parent.AddCommand(project)
 
+	shared.AddCommand(project, List, "list", "Lists all projects with a company management key", func(cmd *cobra.Command) {
```

Comment:
Do we need the `with a company management key`?

Reply:
it won't work with a key limited to specific projects

Reply:
removed

Reply:
yes but this description is a bit weird and not clear.
Maybe:
`Lists all projects (using a company management key)`
or something like that.